source/extensions/filters/http/wasm/wasm_filter.h

```diff
@@ -45,7 +45,6 @@ class FilterConfig : Logger::Loggable<Logger::Id::wasm> {
   }
 
 private:
-  PluginSharedPtr plugin_;
```

Comment:
Would it make sense to leave this and add an assert to verify that `plugin == handle->plugin()`?

Reply:
in order to do that, we have to define == operator on PluginBase and Plugin classes because `plugin_` is the "new" shared_ptr than the one stored in PluginHandle. I think that would make complicates the implementation. wdyt?

Reply:
OK, let's skip it, then.